forage-android/src/main/res/values/dimens.xml

```diff
@@ -6,4 +6,7 @@
     <dimen name="forage_logo_top_margin">0dp</dimen>
     <dimen name="forage_logo_right_margin">0dp</dimen>
     <dimen name="forage_logo_bottom_margin">0dp</dimen>
+    <dimen name="half_vgsfield_padding">0dp</dimen>
```

Comment:
Out of curiosity where are these used?

Reply:
Ha! So this is a really good question. It turns out that these are used by VGS (declared [here](https://github.com/verygoodsecurity/vgs-collect-android/blob/ba63e2680fff975825261da29c9249ea0093b574/vgscollect/src/main/res/values/dimens.xml#L3) and used [here](https://github.com/verygoodsecurity/vgs-collect-android/blob/ba63e2680fff975825261da29c9249ea0093b574/vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/internal/BaseInputField.kt#L152); note that `BaseInputField` is the parent class of `VGSEditText`). 

However, Android's build process creates one giant namespace for all resources. So, what this is doing is it's actually overriding the non-zero values that VGS uses in their code so that we force the padding to zero. It's kinda like CSS in that rules that you put lower in the style sheet take precedence over rules left high in the style sheet. In this case, we, the host app, have to get the final say over what resource values.